src/Algebra/Properties/Ring.agda

```diff
@@ -67,3 +69,23 @@ open AbelianGroupProperties +-abelianGroup public
   - (1# * x)  ≈⟨ -‿cong ( *-identityˡ x ) ⟩
   - x         ∎
 
+x≈0 : ∀ x → x + x ≈ x → x ≈ 0#
```

Comment:
This should be called `x+x≈x⇒x≈0`, otherwise it's a very confusing name! Sounds like every `x` is `0`